include/z64.h

```diff
@@ -654,10 +654,10 @@ typedef struct {
     /* 0x021A */ s16    heartsEnvR[2];
     /* 0x021E */ s16    heartsEnvG[2];
     /* 0x0222 */ s16    heartsEnvB[2];
-    /* 0x0226 */ s16    unk_226;
-    /* 0x0228 */ s16    unk_228;
-    /* 0x022A */ s16    unk_22A;
-    /* 0x022C */ s16    unk_22C;
+    /* 0x0226 */ s16    unk_226; // Used only in uncalled functions
+    /* 0x0228 */ s16    unk_228; // Used only in uncalled functions
+    /* 0x022A */ s16    beatingHeartOscillator;
+    /* 0x022C */ s16    beatingHeartOscillatorDirection; // 1 for decreasing, 0 for increasing 
```

Comment:
alternatively instead of `Direction` you could consider `beatingHeartOscillatorDirection` as a boolean and name it `beatingHeartOscillatorDecrease`, then use true/false to set it and remove `!= 0` in the conditionals.

Removing `!= 0` in conditionals usually doesn't break matching (doesn't change the mips the compiler produces) but sometimes it does

Either way I don't have a strong preference for 0/1-Direction or boolean-Decreasing, just wanted to show that possibility

Reply:
I think I would prefer direction and not treat it as a bool, personally
dont think the word oscillator needs to be in the var though, `beatingHeartDir` is sufficient for me. with the comment explaining which number is which direction

Reply:
I kinda liked @Dragorn421 approach. It makes the code more readable.